cppSettings.gradle

```diff
@@ -72,6 +72,8 @@ task clean(type: Delete) {
     delete buildDir
 }
 
+ext.toolChainPath = null
```

Comment:
An if-not-set check is not wise here. If it is not set, it relies on searching the system path. Looking down the buildscript you will see it is only set in the toolchain if it's set, otherwise it relies on the standard system path. Furthermore, it is only added to the front of `binTools` if it is set. It is impossible to determine a 'default' as this will change on a user-to-user basis, and setting the `toolChain.path` to a blank string will likely break the build logic